### DISCUSSION TITLE
Automating and Provisioning bots with Dockers and Kubernetes

### DISCUSSION THREAD

**Discussion by PolonaS** [whittlem/pycryptobot#762] [Q&A]
Hi I am new on this platform and in the field of crypto trading in general. I am interested in automating the trading process. Where can I add api keys for binance in the code? Additionally, is it possible to run multiple bots on various crypto tickers locally or do you need to use dockers (so one ticker per docker.) and kubernetes?

**Comment by whittlem** [comment]
I've actually documented this in detail on Medium.

Install and Setup of PyCryptoBot 7
https://trading-data-analysis.pro/install-and-setup-of-pycryptobot-7-f1b2c832e795

Please don't forget to clap for the article :)
